docs/ReleaseInstructions.md

```diff
@@ -0,0 +1,27 @@
+### Release process
+
+* Update version `title` in `root_index.html`.
```

Comment:
We will need to somehow lock all released submodules to only allow installations of the other submodules in the same version.

For example,
in v0.1 release, autogluon.tabular==0.1 must have 'autogluon.core==0.1' in setup.py, otherwise release 0.1 could break when we release 0.2 since autogluon.tabular==0.1 would then install autogluon.core==0.2

Reply:
This might need to be done at the time of PyPi release, that way it isn't manual. Somehow we would need to edit the setup.py files specifically in the PyPi release, otherwise pre-releases might also suffer from the same issue (0.2 tabular pre-release installing 0.1 core)

Reply:
Alternatively we can lock version of the dependencies to the same version as the current library

Reply:
that is genius actually, I didn't think about the fact that we would know the version of the package during setup.py

Reply:
about 100x easier than what I was thinking ;)

Reply:
done

Reply:
You'll need to rebase with mainline, I updated the setup.py files recently

Reply:
Done